docs/nidcpower/installation.inc

```diff
@@ -8,7 +8,7 @@ As a prerequisite to using the nidcpower module, you must install the NI-DCPower
 
 The nimi-python modules (i.e. for **NI-DCPower**) can be installed with `pip <http://pypi.python.org/pypi/pip>`_::
 
-  $ python -m pip install nidcpower
+  $ python -m pip install nidcpower~=1.1.5
```

Comment:
Why change this? `pip install` will install latest unless otherwise noted. That's what we want clients to usually do.

Reply:
This is used on PyPI's project description page, which is versioned. So, we want the installation instructions to point to a specific version.